openff/toolkit/topology/molecule.py

```diff
@@ -88,6 +91,9 @@
 #       Only support OEAroModel_MDL in RDKit version?
 
 
+TKR: TypeAlias = Union[ToolkitRegistry, ToolkitWrapper]
```

Comment:
As I understand it, this is as simple as "hey, I'm re-using this type annotation a bunch of times, can I define it as variable-like thing so it's not so repetitive and verbose?"

https://peps.python.org/pep-0613/